src/functions/menus/filterMenusByLocation.js

```diff
@@ -1,28 +1,27 @@
-import formatHeirarchialMenu from "./formatHeirarchialMenu";
+import { camelCase } from "lodash";
 
+import formatHierarchicalMenu from "@/pesayetu/functions/menus/formatHeirarchialMenu";
 /**
  * Filter menus array by menu location.
  *
- * @param  {Array}  menus     The array of WP menus to filter.
- * @param  {Array}  locations The array of locations for filtering.
- * @return {object}           An object containing the requested locations as individual objects.
+ * @author WebDevStudios
+ * @param {Array} menus     The array of WP menus to filter.
+ * @param {Array} locations The array of locations for filtering.
+ * @return {object}         An object containing the requested locations as individual objects.
  */
 export default function filterMenusByLocation(menus, locations) {
   const data = {};
-
   // Loop each menu location.
   locations.forEach((location) => {
     // Convert dashes to underscores.
     const locationName = location.replace(/-/g, "_");
-
     // Filter menus array by location and assign to new object.
-    const wpmenu = menus.filter((menu) => {
+    const wpmenu = menus?.filter(function filterMenu(menu) {
       return menu.locations.includes(locationName.toUpperCase());
     });
-
+    const camelLocation = camelCase(locationName);
```

Comment:
`camelLocation`? Since where are we looking for camels?

I think we should convert the name to camel case where we're declaring it i.e.
```js
 const locationName = camelCase(location.replace(/-/g, "_"));
```

Unless I'm missing what the intention is.

Reply:
@kilemensi Doing this returns this error. I think we should leave it as is 
<img width="544" alt="Screenshot 2021-08-02 at 16 41 36" src="https://user-images.githubusercontent.com/3693563/127971665-4827cfb6-0f80-43ab-ae59-ae85561625fc.png">